.github/ISSUE_TEMPLATE/01_channels_add.yml

```diff
@@ -13,7 +13,7 @@ body:
     id: name
     attributes:
       label: Channel Name
-      description: "Official channel name in English or call sign. May include: `a-z`, `0-9`, `space`, `-`, `!`, `:`, `&`, `.`, `+`, `'`, `/`, `»`, `#`, `%`, `°`, `$`, `@`, `?`, `|`, `¡`"
+      description: "Official channel name in Latin letters or call sign. May include: `a-z`, `0-9`, `space`, `-`, `!`, `:`, `&`, `.`, `+`, `'`, `/`, `»`, `#`, `%`, `°`, `$`, `@`, `?`, `|`, `¡`"
```

Comment:
In the case of Japanese channels, there's a specific term for translating Kanjis/Hiraganas to Latin letters, which is called "romanizing".

Should that be added ? https://en.wikipedia.org/wiki/Romanization